### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by zombiehoffa** [jonaswinkler/paperless-ng#1479] [General]
Is this project dead? No  commits since sept 14 and a large backlog of pull requests makes me a little worried I backed the wrong horse for my document retention needs and would like to know what's going on so I can plan migration if needed.

**Comment by mpfl** [comment]
Chill, mate. 14 September was only three months ago. The developer's a volunteer.

**Comment by Sblop** [comment]
I also fear this project is dead for now. Dev is almost not accessible anymore and does not respond to direct requests. Hopefully someone else is willing to take up this fantastic project…

**Comment by frank-from-cologne** [comment]
I am not a developer, I am a user.
In this aspect, I am interested and willing to do what can be done to keep the project alive.
Perhaps it would also be helpful if the project were carried by several more shoulders - or is that unusual?

Is there anything I can do or that can be done?

Search, call, collect? Or is something like this entirely dependent on someone raising his hand and getting to work, out of himselve, so to speak?

**Reply by TheFehr** [reply]
It is definitly not uncommen for such a project to be carried by more than one person.
The best way forward would be to get the current owners approval on moving the project to an organization and adding multiple users with high enough level of access such as that the triage of issues and pull request can be done by more than just one person.

**Reply by frank-from-cologne** [reply]
Hey @jonaswinkler ,

would that be in your favour, too?
Have you perhaps already thought about this?

BEST REGARDS – this is a great project !!!

**Reply by ghost** [reply]
Paperless-ngx has been created in response to this.

I've created an issue about it, hopefully, @jonaswinkler will respond

**Comment by Sblop** [comment]
I would also be happy to help, managing pull requests and update docker etc.

**Comment by shamoon** [comment]
I Don’t know him personally but for some reason I believe he’ll be back. Jonas put a huge amount of work into this project and pushed hard to take the mantle from paperless. He’s always been open to help from others (including me). But to answer others questions since he’s the only maintainer on the repo at the moment you would have to fork this project unless he changes that.

Anyway, I guess I’m just hoping he’ll be back, and that alls ok…

**Comment by frank-from-cologne** [comment]
On the first hand, I hope Jonas is well, of course 🙌 

Beyond that, I just wanted to suggest taking the burden off his shoulders alone. I never had a fork in mind, and it would be neither purposeful nor effective.

PAPERLESS-NG is alive … and should be for long!

**Reply by torwag** [reply]
People misunderstand the idea of a fork.... a fork is nothing bad, it is one of the main ways to contribute to open source projects. If a group forks paperless-ng to contribute (as a group), it could be moved upstream later. Sure it would be the best, if Jonas is part of that group. If he can't, I believe it would be the best for the project to create a fork and go on.
That said, this group should really make sure they are willing and devoted to put in time and effort to go along. It would not be helpful to have just another dead fork....
Beside of that, I tried to check (stalked) Jonas, as I am also located in Germany. It is hard to say, I hope he is doing well. However, the company who he is working for, did not update the webpages either for a longer period.
Let's hope he can leave a message soon.

**Reply by frank-from-cologne** [reply]
Hi Torwag, yes, and thanks for your explanations. I can follow and see it that way. Including the hope that Jonas will get in touch soon

**Comment by zombiehoffa** [comment]
2 more months with zero commits anywhere and a growing backlog of PRs and issues. You guys still optimistic this thing isn't dead?

**Reply by shamoon** [reply]
you may want to look at https://github.com/paperless-ngx/paperless-ngx
A bunch of us are working on it...
=)

**Comment by natrius** [comment]
It seems like jonaswikler is doing 'something' in there https://github.com/jonaswinkler/paperless-ng/tree/l10n_dev Just so you guys know.

**Reply by shamoon** [reply]
Those are automatically generated by crowdin using his account. Hope alls well with him but still awol

**Comment by NetHans** [comment]
I became aware of this project yesterday and find it very interesting, so I would like to use this. 

Of course, I am fully aware that this project is developed on a voluntary basis and there is no guarantee if and when further developments will be made. 

For me, who is thinking about using this project, it would be very interesting to know, if the developers will continue to work on this project in the future, or if the current status is the end of the development of the project. 

A short statement from the main developers on the subject would be nice and would give me as a user a good feeling when using the software.

Thanks a lot!

**Reply by minideezel** [reply]
Alive and well at https://github.com/paperless-ngx/paperless-ngx

**Reply by smseidl** [reply]
FYI - there is a nice write up of Paperless-NGX in this issue: https://github.com/jonaswinkler/paperless-ng/issues/1632.
